### PR TITLE
PSTRESS-79 Removed a typo error in the variable name

### PR DESCRIPTION
**Pull request by mohitj1988**
Renamed the variable $SIG -> $SIGNAL

### CODE REVIEW

File: pstress/pstress-run.sh [hunk 1]
```diff
@@ -90,7 +90,7 @@ echoit(){
 # Kill the server
 kill_server(){
   SIG=$1
```
Comment (rahulmalik87) [inline pstress/pstress-run.sh:92]: use local

File: pstress/pstress-run.sh [hunk 2]
```diff
@@ -90,7 +90,7 @@ echoit(){
 # Kill the server
```
Comment (rahulmalik87) [inline pstress/pstress-run.sh:90]: can you document the parameter we are passing.

**Review by rahulmalik87 — CHANGES_REQUESTED** [review]

File: pstress/pstress-run.sh (hunk 1 above)
Comment (mohitj1988) [reply pstress/pstress-run.sh:92]: Fixed

File: pstress/pstress-run.sh (hunk 2 above)
Comment (mohitj1988) [reply pstress/pstress-run.sh:90]: Fixed

**Review by rahulmalik87 — COMMENTED** [review]
LGTM

**Pull request merged**